templates/vue/src/components/SettingsModal.vue

```diff
@@ -74,10 +149,30 @@ export default {
       backgroundUrl: "",
       autoLayout: false,
       nodeDraggable: true,
+      defaultPermissions: {
+        public: ["read"],
+        authenticated: ["read"],
+      },
+      permissionsOrder: [
+        "public",
+        "authenticated",
+        ...Object.keys(wpData.roles).filter(
+          role => role !== "administrator" && role !== "author"
+        ),
+      ],
```

Comment:
Can you please pass these two as props into this component (from `Tapestry.vue`) similar to how `permissionsOrder` is passed into NodeModal? That way we only have to set it in one place.